<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parts Inventory Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>1211</width>
      <height>521</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1211</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Century Gothic&quot;;
</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="InventoryDetails">
     <attribute name="title">
      <string>Inventory Details</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>821</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Search of references with count level lower or equal to:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>1151</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Century Gothic&quot;;
background-color: rgb(200, 200, 200);</string>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Reference</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Part Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Min Area</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Max Area</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>No. of Holes</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Min Diameter</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Max Diameter</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
     </widget>
     <widget class="QSpinBox" name="count_filter_txt">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>1</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>3</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="InventoryStatistics">
     <attribute name="title">
      <string>Inventory Statistics</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of parts type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Min Number of Holes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>90</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Number of Holes</string>
      </property>
     </widget>
     <widget class="QLabel" name="LBL_REF_NBR">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Century Gothic&quot;;
color: rgb(100, 100, 255)</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_parts_nmbr">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Century Gothic&quot;;
color: rgb(100, 100, 255)</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_holes2">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>90</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Century Gothic&quot;;
color: rgb(100, 100, 255)</string>
      </property>
      <property name="text">
       <string>REF</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_min_holes2">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Century Gothic&quot;;
color: rgb(100, 100, 255)</string>
      </property>
      <property name="text">
       <string>REF</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_holes">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>90</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Century Gothic&quot;;
color: rgb(100, 100, 255)</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_min_holes">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Century Gothic&quot;;
color: rgb(100, 100, 255)</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>851</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Century Gothic&quot;;
background-color: rgb(200, 200, 200);</string>
      </property>
      <column>
       <property name="text">
        <string>Reference</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Part Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>641</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Top 3 Reference with minimum inventory level</string>
      </property>
     </widget>
     <widget class="QPushButton" name="check_btn">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>230</y>
        <width>171</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(158, 254, 255);
background-color: rgb(147, 147, 147);
font: 18pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Check</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="EditInventory">
     <attribute name="title">
      <string>Edit Inventory</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Part Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Min Area</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Area</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>130</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Holes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>180</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Min Diameter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>230</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Diameter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>280</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>55</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="reference">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>131</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="min_area">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="max_area">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>280</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="max_diameter">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>230</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="min_diameter">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>180</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="number_of_holes">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>130</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="part_name">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="count">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>281</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="update_btn">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>380</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(158, 254, 255);
background-color: rgb(147, 147, 147);</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_btn">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>380</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(158, 254, 255);
background-color: rgb(147, 147, 147);</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_btn">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>380</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(158, 254, 255);
background-color: rgb(147, 147, 147);</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="display_btn">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>46</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(158, 254, 255);
background-color: rgb(147, 147, 147);</string>
      </property>
      <property name="text">
       <string>Display</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>113</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>1221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 30pt &quot;Century Gothic&quot;;
color: rgb(158, 254, 255);
background-color: rgb(147, 147, 147);</string>
    </property>
    <property name="text">
     <string>Parts Inventory Manager</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>690</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Century Gothic&quot;;
color: rgb(158, 254, 255);
background-color: rgb(147, 147, 147);</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1262</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
